### ISSUE TITLE
[feat] Dark theme implementation

### ISSUE THREAD

**Issue by beradeep** [beradeep/CollegeSearch#2]
_Implement night/dark theme colors and UI for the app._

**Comment by abhay-ajith** [comment]
Hi,
Is the issue to change the theme to dark or being able to toggle between.

**Comment by beradeep** [comment]
@abhay-ajith It is not an issue. The `dark theme` needs to be implemented as a new feature by creating a separate `darkColorPalette` for the dark theme. Having the option to toggle between `light` and `dark` themes is a plus.

**Comment by JonibekXolmonov** [comment]
Are the colors inside _darkColorScheme(now commented) real colors of dark mode?

**Comment by beradeep** [comment]
Yes, they were actually supposed to be the colors for dark mode. But most parts of the codebase right now has colors hardcoded only for light mode, for that reason dark mode is disabled for now. These need to be replaced with colors from the colorSchemes using `MaterialTheme.colorScheme`, for consistency in both the modes.

**Comment by JonibekXolmonov** [comment]
I give it a try.

**Comment by beradeep** [comment]
> I give it a try. 

Hi, are you working on it?

**Comment by JonibekXolmonov** [comment]
Hi, since you assigned me, I have been stuck in university tasks and then 2 day holiday, from tomorrow I begin working. I am so sorry to be late

**Comment by beradeep** [comment]
Hi, no need to be sorry 😄, thanks for informing!

**Comment by JonibekXolmonov** [comment]
🤝🤝

**Comment by beradeep** [comment]
Hi, are you still working on this?

**Comment by JonibekXolmonov** [comment]
Hi sir, can you unassign me? Because of my studies at the university,diploma thesis and prior-graduation practice, I can't allocate time even to breathe. 
As soon as I am free, I let you know and go on enhancing your project. I am really sorry not informing this earlier

**Comment by beradeep** [comment]
Hi, it's okay. I just wanted to know if there's any update. As per your request, I am unassigning you right now. Just let me know if you start working on this again.

**Comment by JonibekXolmonov** [comment]
Sure,🤝

**Comment by amanjn38** [comment]
@beradeep Is this done ? Or can I pick this up ?

**Comment by beradeep** [comment]
@amanjn38 go ahead 👍

**Comment by amanjn38** [comment]
@beradeep The PR for dark theme implementation
https://github.com/beradeep/CollegeSearch/pull/5

**Comment by ShashwatGohel** [comment]
hello 
is any one working on this project, or if any other feature is to be implemented in the app
